app/api/files/specs/jsRoutes.spec.js

```diff
@@ -156,27 +156,44 @@ describe('upload routes', () => {
     beforeEach(() => {
       app = express();
       uploadRoutes(app);
+      remoteApp = express();
+      remoteApp.post('/api/public', (_req, res) => {
+        res.json(_req.headers);
+      });
     });
 
     afterEach(async () => {
       await remoteServer.close();
     });
 
     it('should return the captcha and store it', done => {
-      remoteApp = express();
-      remoteApp.post('/api/public', (_req, res) => {
-        res.json(_req.headers);
+      remoteServer = remoteApp.listen(54321, async () => {
+        const response = await request(app)
+          .post('/api/remotepublic')
+          .send({ title: 'Title' })
+          .set('tenant', 'tenant')
+          .expect(200);
+
+        const headersOnRemote = JSON.parse(response.text);
+        expect(headersOnRemote.tenant).not.toBeDefined();
+        done();
```

Comment:
@mfacar the description does not match what this is testing.

Reply:
I unified with the previous test and updated the description